modules/bitgo/test/unit/bitgo.ts

```diff
@@ -81,14 +81,8 @@ describe('BitGo Prototype Methods', function () {
     it('throws if HMAC request verification is disabled for non-dev environments', function () {
       (() => TestBitGo.decorate(BitGo, { env: 'prod', hmacVerification: false }))
         .should.throw(/Cannot disable request HMAC verification in environment/);
-      (() => TestBitGo.decorate(BitGo, { env: 'test', hmacVerification: false }))
-        .should.throw(/Cannot disable request HMAC verification in environment/);
```

Comment:
instead of deleting, can we just change the assertion to call `.should.not.throw` ?